src/Microsoft.Azure.Batch.SoftwareEntitlement.Server/Model/AcquireRequestBody.cs

```diff
@@ -0,0 +1,24 @@
+namespace Microsoft.Azure.Batch.SoftwareEntitlement.Server.Model
+{
+    /// <summary>
+    /// A request for a leased software entitlement acquisition
+    /// </summary>
+    /// <remarks>Used to rehydrate the JSON body included with the POST requesting an entitlement.</remarks>
+    public class AcquireRequestBody : IVerificationRequestBody
```

Comment:
Is deserialization of this case sensitive? I thought the JSON payload specified `token`, `applicationId` etc ...